src/profile.js

```diff
@@ -244,7 +244,9 @@ var Profile = CreateClass({
         }
         if (e.target.type === "checkbox") {
             newState[key] = e.target.checked;
-            newState.filterOptionsChanged = true;
+            if (key.toLocaleString().toLowerCase().includes("rule")) {
```

Comment:
toLocaleLowerCase() ? or isn't it just match "ruleMaxSize" ?

Reply:
I thought in the future we might add new rules so it checks "rule" keyword as a whole. I agree toLocateString is probably unnecessary and will revise it.

Reply:
Updated in 47ad220

Reply:
- .startsWith("rule")
  - (old IE does not support the method, other browsers may require polyfill)
- regex is better and flexible if the key matching rule became complex

as it already changed, "includes" is ok if no unexpected match cases.

- [x] searched "rule" may become a part of another word in wordlist
  - results are only: "rule", "overrule"  it's a rare case, currently no need to care.
  - other chance is when want to use "rule" in another compound words.
    we will need strict matching pattern, but ok not now.